docs/buildah-run.md

```diff
@@ -31,8 +31,9 @@ runtime, the manpage to consult is `runc(8)`)
 **--tty**
 
 By default a pseudo-TTY is allocated only when buildah's standard input is
-attached to a pseudo-TTY. Setting the `--tty` option to `true` will cause a
-pseudo-TTY to be allocated inside the container. Setting the `--tty` option to
+attached to a pseudo-TTY.  Setting the `--tty` option to `true` will cause a
+pseudo-TTY to be allocated inside the container connecting the user's "terminal"
+with the stdin and stdout stream of the container.  Setting the `--tty` option to
```

Comment:
How about we add.

if you execute buildah run and expect an interactive shell, you need to specify the --tty flag.

Reply:
@rhatdan  Good thought, added to the end of the description.